### ISSUE TITLE
Full Site Editing: plugin data inserted multiple times 

### ISSUE THREAD

**Issue by vindl** [Automattic/wp-calypso#34415]
**Note: hold for onboarding, please do not self-assign this issue**

The Full Site Editing plugin currently creates default data on plugin activation. That allows us to power header and footer overwrites during front end rendering and editing.

https://github.com/Automattic/wp-calypso/blob/151293d3fa2b53b8e6362d3beb99086dc866d695/apps/full-site-editing/full-site-editing-plugin/full-site-editing-plugin.php#L106

If you follow the instructions available here PCYsg-ly5-p2 to set up the plugin locally, after activating the plugin you should see the following CPTs created and populated:

<img width="511" alt="screenshot-2019-06-12-at-17 46 11" src="https://user-images.githubusercontent.com/1182160/60530596-62ce7c80-9cf9-11e9-9ce6-df3d88b05e73.png">
<img width="516" alt="screenshot-2019-06-12-at-17 46 25" src="https://user-images.githubusercontent.com/1182160/60530598-62ce7c80-9cf9-11e9-85a6-95daa04e6762.png">

You can access these if you click on `Templates` and `Template Parts` items in wp-admin sidebar. If you deactivate and activate the plugin again, these posts will be duplicated. We need to propose a proper way to handle this and implement it in order to close this issue.

Possible follow up tasks:
  - investigate whether this issue and the fix we come up with works with Simple and Atomic sites. If it doesn't create separate issues for those. 

/cc @mattwiebe

**Comment by mattwiebe** [comment]
tl;dr I'd suggest closing this as a `wontfix` (or `maybelater`) - these are just conveniences for local development and won't really have a bearing on the way the plugin runs on Simple or Atomic sites.

---

Most plugins I've seen will set an option during activation and check for it and bail early in the activation routine if it's there. Often it will just record something like the current plugin version number, so that the plugin can also check for any subsequent updates it would need to do in the future, if some data structures need to be migrated or changed in some fashion.

On a practical note, it's unlikely that we really need to think about this. The plugin won't be truly "activated" on Atomic sites (it will be directly loaded through `mu-plugins`) and the same is true on Simple sites, where content population will happen through Headstart. And, in actuality, all Atomic sites start first as Simple sites that are subsequently migrated.

**Comment by vindl** [comment]
> tl;dr I'd suggest closing this as a wontfix (or maybelater) - these are just conveniences for local development and won't really have a bearing on the way the plugin runs on Simple or Atomic sites.

Agreed, I just thought it might be a good first task to get the local development going and get a sense of overall dev flows.

**Comment by vindl** [comment]
Resolved in https://github.com/Automattic/wp-calypso/pull/34566 but we are holding off on merging it for now.
